index.html

```diff
@@ -1,56 +1,147 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Справжня ікра</title>
-
-    <link rel="stylesheet" href="node_modules/reset-css/reset.css" />
-    <link rel="stylesheet" type="text/css" href="src/scss/main.scss" />
-  </head>
-  <body>
-    <div class="test century">
-      <p class="century">Hello, world!<br /></p>
-      <p class="century">And hello peaple!</p>
-    </div>
-    <div class="test">
-      <div>
-        <p class="gilroy fw-semibold">Gilroy Medium (600)</p>
-      </div>
-      <div>
-        <p class="gilroy">Gilroy Regular (400)</p>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Справжня ікра</title>
+
+  <link rel="stylesheet" href="node_modules/reset-css/reset.css" />
+  <link rel="stylesheet" type="text/css" href="src/scss/main.scss" />
+</head>
+
+<body>
+  <header id="header-page" class="header-page">
+    <nav class="container-fluid navbar navbar-expand-lg">
+      <div class="container gilroy fw-light text-6">
+        <a class="navbar-brand col-4" href="#">
+          <img class="img-logo-adaptiv" src="src/assets/icons/logo.svg" alt="Logo-company" />
+        </a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
+          aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
```

Comment:
`aria-*` need for accessibility, remove this.
```suggestion
        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent">
          <span class="navbar-toggler-icon"></span>
        </button>
```